reports/utils.go

```diff
@@ -1,11 +1,55 @@
 package reports
 
 import (
+	"bytes"
+	"encoding/json"
+	"errors"
 	"fmt"
+	"io"
+	"net/http"
+	"strings"
 
 	"github.com/trackit/trackit-server/aws"
 )
 
+func mergeStringJson(style1 string, style2 string) (string, error) {
+	merged := make(map[string]interface{})
+	err := json.NewDecoder(strings.NewReader(style1)).Decode(&merged)
+	if err != nil {
+		return "", nil
+	}
+	err = json.NewDecoder(strings.NewReader(style2)).Decode(&merged)
+	if err != nil {
+		return "", nil
+	}
+	output, err := json.Marshal(merged)
+	if err != nil {
+		return "", nil
+	}
+	return string(output), nil
+}
+
+func getAwsAccount(account string, aas []aws.AwsAccount) *aws.AwsAccount {
+	for _, aa := range aas {
+		if aa.AwsIdentity == account {
+			return &aa
+		}
+	}
+	return nil
+}
+
+func formatAwsAccount(aa aws.AwsAccount) string {
+	return fmt.Sprintf("%s (%s)", aa.Pretty, aa.AwsIdentity)
+}
+
+func getAwsIdentities(aas []aws.AwsAccount) []string {
```

Comment:
Are those three function not available somewhere else in the codebase ?

Reply:
These functions are not available somewhere else.